src/lib/evil/unposix/sys/types.h

```diff
@@ -1,11 +1,27 @@
 #ifndef UNPOSIX_SYS_TYPES_H
 #define UNPOSIX_SYS_TYPES_H
 
-#include <stdint.h>
+#ifndef WIN32_LEAN_AND_MEAN
+# define WIN32_LEAN_AND_MEAN
+#endif
 
 #ifdef _MSC_VER
```

Comment:
I think it would make more sense to let this `ifndef` inside `#ifdef _MSC_VER`, but to be honest I can't think why I've previously let that `_MSC_VER`: this header exists only when compiling for Windows. Hm.

Reply:
Good thing I've slept. Now that I'm awake that I remembered Evil isn't just for Windows, but for MacOS too.

Reply:
Wouldn't be batter to put all the files on a #ifdef then? like:
#ifndef UNPOSIX_<dependence>

Reply:
That would be better when we reach MacOS. Since we don't know what on Mac is not implemented in Windows, we don't know which missing parts do intersect.